packages/docs/.gitignore

```diff
@@ -7,6 +7,7 @@
 # Generated files
 .docusaurus
 .cache-loader
+/static/api/index.html
```

Comment:
Is this html auto generated?

Reply:
Yes. Generated by `redoc-cli build https://logto.dev/api/swagger.json -o static/api/index.html --title 'Logto Core API Reference'`

---

Reference: see the PR summary

- Generate the static html file of API Reference according to the specification https://logto.dev/api/swagger.json
    - Should put the HTML file in the directory /static/* for storing static resources that need to be controlled by Docusaurus.
        - https://github.com/facebook/docusaurus/issues/3894

Reply:
ok, I see